R/ops.R

```diff
@@ -0,0 +1,44 @@
+#' @export
+ops <- structure(list(), class = "torch_ops")
```

Comment:
I like the name `ops` but it might cause issues with namespacing. What do you think about renaming this object to `jit_ops`?

Reply:
Yeah, I also thought `ops` might be too general ... `jit_ops` is great!